### ISSUE TITLE
v-row & v-col

### ISSUE THREAD

**Issue by eschirtz** [vuetifyjs/eslint-plugin-vuetify#6]
Hey, trying to upgrade to Vuetify 2.0.0 and I've installed `eslint-plugin-vuetify` via npm. When I add the rules `vuetify/no-legacy-grid`, my v-flex/v-layout tags are corrected to use `v-col` and `v-row`. However, I can't seem to find anything about these two new tags in the docs? Is this a future feature that hasn't yet been implemented in the latest release of Vuetify? 

Thanks!

**Comment by eschirtz** [comment]
Just saw the footnote about documentation not being updated. I will look in the API explorer.

**Comment by KaelWD** [comment]
Yeah it's been implemented, I just wrote this plugin before documentation so it's all a bit hard to actually use right now. The previous grid system will keep working if you just disable the two grid rules, and you can turn them back on once the documentation is done.

**Comment by eschirtz** [comment]
That makes total sense, so far the plug-in has been working like a charm. Great work!

**Comment by husayt** [comment]
I lost all paddings and margins with V-row and V-col and there is no single example or documentation I could refer to in order to find a solution

**Comment by wubzz** [comment]
Same issue here, paddings and margins are way of compared to `v-layout` / `v-flex`. Assuming this will be fixed by the time it reaches official Vuetify documentation. Until then I'd recommend sticking with 'legacy'.

**Comment by eschirtz** [comment]
One thing you should be sure to do is use the Regex expressions in the migration guide to change the margins/paddings to their new equivalent values. When I first switched my margins/paddings we're all off, but after I did find and replace for all three or so Regex expressions they provided, everything went back to normal!

**Comment by horaciosolorio** [comment]
I use eslint-plugin-vuetify to update from 1.5 and all was replaced but for the below  code don't

<v-layout
        column
        align-center>

and linter show error in "column".

I replace with:
<v-col aling-self="center">

But my component is broken , i don't understand what is the correct way to use the new grind system, could you help me with some example , still i can see de documentation updated

I was using the column approche in all my proyect and now its broken.
Can you help please.

**Comment by BenRichter** [comment]
@horaciosolorio v-layout must be replaced by **v-row**. 
If you search in https://vuetifyjs.com/en/components/api-explorer for v-col you find a list with its props, but without description yet. I guess the column direction has been dropped because it's not part of the bootstrap 4 grid.
